Frontend/src/components/forms/partial/CustomTextField.jsx

```diff
@@ -20,6 +19,11 @@ export default function TextInput({
   function handleChange(e) {
     const inputValue = e.target.value;
 
+    // disable/remove leading whitespaces
+    if (inputValue === " ") {
+      inputValue = inputValue.trim();
```

Comment:
Zašto uvijek ne trimamo?

Reply:
Jer onda ne mozes upisati razmak u polje, tj razmak se automatski obrise.